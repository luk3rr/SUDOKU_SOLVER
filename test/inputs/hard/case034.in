000000090 085093002 000860000 000050030 050006070 002080019 100500004 043000005 000700021 
