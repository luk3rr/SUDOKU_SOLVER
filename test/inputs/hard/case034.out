Solving the following grid:
0 0 0 | 0 0 0 | 0 9 0 
0 8 5 | 0 9 3 | 0 0 2 
0 0 0 | 8 6 0 | 0 0 0 
------+-------+------
0 0 0 | 0 5 0 | 0 3 0 
0 5 0 | 0 0 6 | 0 7 0 
0 0 2 | 0 8 0 | 0 1 9 
------+-------+------
1 0 0 | 5 0 0 | 0 0 4 
0 4 3 | 0 0 0 | 0 0 5 
0 0 0 | 7 0 0 | 0 2 1 


Solution found :')

2 1 6 | 4 7 5 | 8 9 3 
7 8 5 | 1 9 3 | 6 4 2 
4 3 9 | 8 6 2 | 1 5 7 
------+-------+------
8 9 1 | 2 5 7 | 4 3 6 
3 5 4 | 9 1 6 | 2 7 8 
6 7 2 | 3 8 4 | 5 1 9 
------+-------+------
1 2 7 | 5 3 8 | 9 6 4 
9 4 3 | 6 2 1 | 7 8 5 
5 6 8 | 7 4 9 | 3 2 1 

Algorithm: BFS
Total time: 1078 ms
Total expanded states: 167507
